mne/viz/tests/test_epochs.py

```diff
@@ -72,23 +74,17 @@ def _get_epochs_delayed_ssp():
     return epochs_delayed_ssp
 
 
-def test_plot_epochs():
-    """ Test plotting epochs
-    """
+def test_plot_trellis():
+    """ Test plotting epochs using Trellis plot"""
     import matplotlib.pyplot as plt
     epochs = _get_epochs()
     epochs.plot([0, 1], picks=[0, 2, 3], scalings=None, title_str='%s')
     plt.close('all')
     epochs[0].plot(picks=[0, 2, 3], scalings=None, title_str='%s')
     plt.close('all')
-    # test clicking: should increase coverage on
-    # 3200-3226, 3235, 3237, 3239-3242, 3245-3255, 3260-3280
-    fig = plt.gcf()
-    fig.canvas.button_press_event(10, 10, 'left')
```

Comment:
You mean why I removed them? It was left there by accident. I don't think trellis plot handles left key at all.